doc/index.rst

```diff
@@ -263,24 +263,34 @@ ABI checks
 
 Generally speaking, the `ABI <https://en.wikipedia.org/wiki/Application_binary_interface>`_ of a library might break for various reasons. A detailed explanation and a list of DOs and DON'Ts can be found in the `KDE Community Wiki <https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B>`_.
 
-The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to a code archive with your stable version (e.g. tagged version or older tagged versions of your package)
+The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to the stable version (i.e. you want to check against with this for sure) of your code. Two types of value can be passed to `ABICHECK_URL`:
+
```

Comment:
There are way more than two types. I don't understand why you need this distinction between tags and branches.

For `ros_canopen` I maintain stable branches for each distro, so I would point the check to a branch-based URL.
The same holds for many ROS-I repositorires.
`moveit`  does not maintain  stable branch, so in this case the maintainer has to specify a tag-based or a commit-based URL to test against a specific baseline. Or against the upstream branch of the release repo or to an older development branch which is considered stable. Or the source code gets uploaded manually somewhere.

Reply:
> I don't understand why you need this distinction between tags and branches.

So that it'd be easier to understand.

> There are way more than two types.

In my way of distinction there are still only 2.

> For ros_canopen I maintain stable branches for each distro, so I would point the check to a branch-based URL.

branch

> moveit does not maintain stable branches, so in this case the maintainer has to specify a tag-based or a commit-based URL to test against a specific baseline.

tag

> Or against the upstream branch of the release repo or to an older development branch which is considered stable.

branch

Reply:
And what about https://github.com/ros-planning/moveit/archive/7d36403187dffbad7a747e65b98dd266cc0b9da9.zip?

Or even http://repositories.ros.org/ubuntu/main/pool/main/r/ros-kinetic-joint-state-publisher/ros-kinetic-joint-state-publisher_1.12.8.orig.tar.gz

> So that it'd be easier to understand.

To understand *what*?